boost/1_53/CMakeLists.txt

```diff
@@ -1,34 +0,0 @@
-add_definitions(-DBOOST_PYTHON_NO_PY_SIGNATURES -DBOOST_PYTHON_STATIC_LIB)
-include_directories("${vsUTCS_SOURCE_DIR}/boost/1_53")
-add_library(boost_python
```

Comment:
This is from Boost, you shouldn't be changing it.

Maybe you have to update boost. You can build using system boost libraries, instead of the included ones. I would recommend you try.

Reply:
This file is one of only two files checked into GitHub under the boost directory. It didn't come with the boost source code when I downloaded that.

With this file in place, the entire project failed to build. Without this file, the project built and ran fine. So I decided just to remove this file.

If there is an alternative, I would be happy to hear about it.

Reply:
Oh, I forgot to mention: I did try building using the system boost libraries also. That didn't work either.

Reply:
UPDATE: I have since figured out how to build using the system boost libraries.

Reply:
These boost directories should be git submodules. I think this is a migration error carried over since the migration from SVN. I'll look into that.

In the meanwhile, I think it's ok to delete these.